tools/code-owners-parser/CodeOwnersParser/CodeownersFile.cs

```diff
@@ -0,0 +1,115 @@
+using System;
```

Comment:
FYI: this file was renamed from `CodeOwnersFile` but git didn't pick up on that.